### ISSUE TITLE
nanoc live fails with KeyError

### ISSUE THREAD

**Issue by iay** [nanoc/nanoc#1350]
nanoc 4.9.4 breaks `live` command; it's fine in 4.9.3

### Steps to reproduce

1. install nanoc 4.9.4 (or the current github HEAD)
2. `bundle exec nanoc live`
3. profit

### Expected behavior

Live preview of site available.

### Actual behavior

Exception thrown.

### Details

```
Captain! We’ve been hit!

KeyError: key not found: :host

  0. /Volumes/Cased/git/iay/iay/site/vendor/ruby/2.4.0/gems/nanoc-4.9.4/lib/nanoc/cli/commands/view.rb:34:in `fetch'
  1. /Volumes/Cased/git/iay/iay/site/vendor/ruby/2.4.0/gems/nanoc-4.9.4/lib/nanoc/cli/commands/view.rb:34:in `run'
  2. /Volumes/Cased/git/iay/iay/site/vendor/ruby/2.4.0/gems/guard-nanoc-2.1.3/lib/guard/../nanoc/cli/commands/live.rb:24:in `block in run'
```

There have been a couple of commits in the repository that look like they might be related to this, but the current HEAD has the same behaviour.

### Crash log

https://gist.github.com/iay/6c587d250c03a446c8478257b22130ce

**Comment by ddfreyne** [comment]
Ah yes — the `view` command got refactored, and the host/port/handler options are specified in code in a different way now. Oops!

**Comment by ddfreyne** [comment]
This is fixed in the guard-nanoc v2.1.4 release.

**Comment by iay** [comment]
Confirmed; many thanks.
